### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'parts'

### ISSUE THREAD

**Issue by ptynecki** [smsapi/smsapi-python-client#22]
Hey,

I got a new exception today after `api.sms.send()` execution. I'm using PyPy 7.1.1 (Python 2.7.13) with smsapi-client 2.3.0.

```python
/project/env/site-packages/smsapi/endpoint.pyc in __call(api, **kwargs)
    124     def __call(api, **kwargs):
    125         endpoint = ApiEndpoint(api, kwargs)
--> 126         return endpoint.send_request()
    127
    128     return __call

/project/env/site-packages/smsapi/endpoint.pyc in send_request(self)
     96             raw_response = requests.request(self.method, url, **kwargs)
     97
---> 98             return self.process_response(raw_response, url=url)
     99
    100         def process_response(self, raw_response, url=None):

/project/env/site-packages/smsapi/endpoint.pyc in process_response(self, raw_response, url)
    113             if isinstance(self.response_mapping, tuple):
    114                 model, type = self.response_mapping
--> 115                 response = type.parse(response, model)
    116             elif self.response_mapping:
    117                 response = self.response_mapping.from_dict(response, raw_response=raw_response)

/project/env/site-packages/smsapi/models.pyc in parse(cls, content, model)
     88
     89         for data in content:
---> 90             m = model.from_dict(data)
     91             collection.append(m)
     92

/project/env/site-packages/smsapi/models.pyc in from_dict(cls, data, **kwargs)
     11             args[str(k)] = v
     12
---> 13         return cls(**args)
     14
     15     def __repr__(self):

TypeError: __init__() got an unexpected keyword argument 'parts'
```

**Comment by f-villegas** [comment]
Me too. I am using smsapi-client 2.2.4. **args example now: 
<class 'dict'>: {'submitted_number': 'xxxxxxxx', 'number': 'xxxxxxxx', 'points': 0.033, 'error': None, 'date_sent': 1571150604, 'idx': None, 'id': '5DA5DB0C37383050', 'status': 'QUEUE', 'parts': 1}

**Comment by in-void** [comment]
already resolved in 2.3.1 enjoy
